From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect sound/timer.h include in sound pcm.c
Patch-mainline: never, kabi
References: kabi

In 4.4.129, commit bb256eeaec50d26413822ab8ef1fa51789fa0a2c (ALSA: pcm: Fix
UAF at PCM release via PCM timer access), upstream commit
a820ccbe21e8ce8e86c39cd1d3bc8c7d1cbb949b added sound/timer.h include
to sound/core/pcm.c and it made some of the symbols defined.

Protect the include by __GENKSYMS__ to satisfy the kABI checker.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 sound/core/pcm.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/sound/core/pcm.c
+++ b/sound/core/pcm.c
@@ -28,7 +28,9 @@
 #include <sound/core.h>
 #include <sound/minors.h>
 #include <sound/pcm.h>
+#ifndef __GENKSYMS__
 #include <sound/timer.h>
+#endif
 #include <sound/control.h>
 #include <sound/info.h>
 
